content/en/community/contribution-guidelines.md

```diff
@@ -13,7 +13,12 @@ description: >
 <!-- cSpell:locale en-US -->
 # Contribution Guidelines
 
-Thank you for contributing your time and expertise to CDEvents. This document describes the contribution guidelines for the project.
+Thank you for contributing your time and expertise to CDEvents. This document describes the contribution guidelines for the project specification and website:
+
+* [sdk-go](https://github.com/cdevents/sdk-go?tab=readme-ov-file#contributing)
+* [sdk-java](https://github.com/cdevents/sdk-java?tab=readme-ov-file#contributing)
+
+The CDEvents SDKs have dedicated contributing guides in the respective repositories.
```

Comment:
This list feels a bit misplaced here. Should there instead be a ## section about SDKs here?

Reply:
The text below should be before the list, the links point to the contributing guidelines for the SDKs.
Thanks for noticing this.